maps/away/ships/konyang/kasf_ship/kasf_ship.dm

```diff
@@ -139,6 +139,14 @@
 /obj/machinery/computer/shuttle_control/explore/kasf_shuttle
```

Comment:
this should be using the `/obj/machinery/computer/shuttle_control/explore/terminal` subtype instead of redefining the terminal wars here